kuka_kr270_support/urdf/kr270r2700_macro.xacro

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://wiki.ros.org/xacro">
+
+  <xacro:macro name="kuka_kr270r2700" params="prefix">
+    <xacro:include filename="$(find kuka_resources)/urdf/common_materials.xacro"/>
+    <!-- links -->
+    <link name="${prefix}base_link">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/visual/base_link.dae"/>
+        </geometry>
+        <xacro:material_kuka_pedestal />
```

Comment:
```suggestion
        <xacro:material_kuka_black />
```
sorry for the confusion but `kuka_pedestal` seems to be some sort of dark grey while the base is actually black

Reply:
So, base and link 6 black, and link 1-5 orange?

Reply:
exactly, that matches the pdf quite well

Reply:
Done in 492fd2c72c8306d27eb2032a7526703bf972684b